main_server.py

```
@@ -1155,13 +1167,38 @@ async def proactive_chat(request: Request):
+            for attempt in range(max_retries):
+                try:
+                    response = await asyncio.wait_for(
+                        llm.ainvoke([SystemMessage(content=system_prompt)]),
+                        timeout=10.0
+                    )
+                    response_text = response.content.strip()
+                    break  # 成功则退出重试循环
+                except (RateLimitError, APIError) as e:
+                    if attempt < max_retries - 1:
+                        wait_time = retry_delays[attempt]
+                        logger.warning(f"[{lanlan_name}] 主动搭话LLM调用失败 (尝试 {attempt + 1}/{max_retries})，{wait_time}秒后重试: {e}")
```

Ruff (0.14.6)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)